### ISSUE TITLE
Numpy is installed twice

### ISSUE THREAD

**Issue by ericflores** [DeftWork/rpi-tensorflow#5]
There is one improvement area that I found. At least for numpy, the docker build spents about 15-20 minutes compiling and installing an old version (not sure why, I think it was version 1.11.x), but tensorflow 1.7.0 requires numpy>1.13.3, so the tensorflow installer got to uninstall the recently installed version and install version 1.14.2. Another 20 minutes (probably less). Not a problem when rebuilding, but it could install faster if minimum version of python packages installed ahead of tensorflow are specified to be closer to what the current version of tensorflow requires.  

Tensor flow requisites are down in this page:
https://pypi.python.org/pypi/tensorflow

**Comment by DeftWork** [comment]
I've tried distinct install combinations, but this is the only one that works for me, but you are welcome to colaborate , I'm sure you are able to find the way to install the required version of numpy and many other improvements.

**Comment by ericflores** [comment]
Still working on this, and as you observed, any change can break stuff.
FYI, I found out today that pandas doesn't compile when using pip 10.0.0 (latest version).

Others found the same:
https://github.com/pandas-dev/pandas/issues/20666

In the meantime, 
pip install --upgrade pip==9.0.3
solves the issue while a fix is implemented.

**Comment by DeftWork** [comment]
Yes, you are right, I've rebuilded the dockerfile and it doesn't work, I've aplied your fix and it's working ok.
Thanks you for your advice. I would love you accept my invitation to become a collaborator.
